README.md

```diff
@@ -41,7 +41,7 @@ refer to the following table for info on where to go to be properly assisted.
 
 | Category | Course of action (CHOOSE ONE)|
 | -------- | --------------- |
-| App/theme related issues | <ul><li>Post on our Jira (preferred)</li><li>Post in the [Substratum Google+ community](https://plus.google.com/communities/102261717366580091389)</li><li>Join our [Telegram support channel](https://t.me/SubstratumSupport)</li></ul> |
+| App/theme related issues | <ul><li>Post on our Jira (preferred)</li><li>Post in the [Substratum Google+ community](https://plus.google.com/communities/102261717366580091389)</li><li>Join our [Telegram support channel](https://t.me/joinchat/BTodxkJMTTTgYmCfPL7IIA)</li></ul> |
```

Comment:
Would rather change to say "#support from our [Telegram Link Portal](https://t.me/substratum)"